YeusofGeekyTrail.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <h1 id="title">Yeusof & Geeky Trail</h1>
+    <p id="description">This is our our code</p>
+    <form id="survey-form" action="">
+        <input id="name" type="text" placeholder="Enter your name" required />
+        <input id="email" type="email" placeholder="Enter your email" required />
+        <input id="number" type="number" min="0" max="99999999" placeholder="Enter Phone" required />
+        <label id="name-label">Name</label>
+        <label id="email-label">Email</label>
+        <label id="number-label">Phone Number</label>
+        <select id="dropdown" required>
+        <option value="Yesuf">Yesuf</option>
+        <option value="Geeky">Geeky</option>
+      </select>
+        <label id="attribute"> <input type="radio" value="attribute" name="attribute"/> Yesuf</label>
+        <label id="attribute"> <input type="radio" value="attribute" name="attribute"/>Geeky</label>
+
+        <label id="attribute"> <input type="checkbox" value="attribute" /> attribute</label>
+        <label id="attribute"> <input type="checkbox" value="attribute"/>attribute</label>
+        <textarea name="textarea" id="" cols="30" rows="10"></textarea>
+        <button id="submit" type="submit">Submit</button>
```

Comment:
This project has many id attributes for elements. Do not use an ID selector for multiple elements.  Always use the class attribute for multiple elements.